packages/raygun4ocaml/raygun4ocaml.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+name: "raygun4ocaml"
+version: "0.1.0"
+available: [ ocaml-version >= "4.02.0" ]
+maintainer: "Lidcore <it@lidcore.com>"
+authors: [ "Romain Beauxis" ]
+license: "ISC"
+homepage: "https://github.com/lidcore/raygun4ocaml"
+dev-repo: "https://github.com/lidcore/raygun4ocaml"
```

Comment:
On opam `1.2.0` you'll get:
```
[ERROR] At ~/.opam/repo/default/packages/raygun4ocaml/raygun4ocaml.0.1.0/opam:9:50:
  Not a recognised version-control URL
```
Can you just suffix that url with `.git`? Thanks!